docs/source/deployment/databricks.md

```diff
@@ -263,3 +263,9 @@ Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line mag
 ```ipython
 In [2]: %run_viz
 ```
+
+### How to use Datasets stored on Databricks DBFS
+
+DBFS is a distributed file system mounted into a DataBricks workspace and accessible on a DataBricks cluster. It maps cloud object storage URIs to relative paths so as to simplify the process of persisting files. With DBFS, libraries can read from or write to distributed storage as if it's a local file.
+To use datasets like `SparkDataSet` with DBFS, the file path must be prefixed with `/dbfs`.
```

Comment:
See my comment below, if users want to use DBFS, they should explicitly add the prefix to every file path.

There is a subtlety here, which is that `PySpark` code will natively look in DBFS by default, though as we established in https://github.com/kedro-org/kedro-plugins/pull/114, even if the user wants to use `PySpark` via `SparkDataSet` (which *should* work natively, but doesn't because of the way we coded this data set), they should still specify the `/dbfs` prefix.

```suggestion
To use datasets with DBFS, the file path passed to the dataset **must** be prefixed with `/dbfs/`. This applies to all datasets, including `SparkDataSet`.
```